dinov2/eval/utils.py

```diff
@@ -60,7 +60,7 @@ def evaluate(
     for metric in metrics.values():
         metric = metric.to(device)
 
-    metric_logger = MetricLogger(delimiter="  ")
+    metric_logger = MetricLogger(delimiter="  ", verbose=distributed.get_local_rank() == 0)
```

Comment:
Could be changed to `verbose=distributed.is_main_process()`, but also works as is.